genie/dashboard_table_updater.py

```diff
@@ -151,7 +151,7 @@ def update_cumulative_sample_table(
     seg_counts["Total"] = sum(seg_counts)
 
     total_counts = pd.DataFrame(clinical_counts)
-    total_counts["Fusions"] = fusion_counts
+    total_counts["Fusions"] = sv_counts
```

Comment:
Any reason we don't update the `total_counts["Fusions"]` to `total_counts["Structural_Variants"]` or something like that? Similar question for Lines 266, 459, 465?

Reply:
@rxu17 That is a great question.  We should create a ticket to rename "Fusions" to "Structural_Variants", BUT... All fusions are structural variations and vice versa.

Reply:
Makes sense!
I just wasn't sure if it mattered that it affected the appearance of the `cumulative_sample_table` in synapse in that it still used `Fusions` as the naming convention

Reply:
Ya... the tables themselves still use `Fusions` as the column header so thats why. We would need to update the headers in the Synapse tables along with the code which is why those are the same.